patches/minecraft/net/minecraft/server/MinecraftServer.java.patch

```diff
@@ -1,14 +1,42 @@
 --- a/net/minecraft/server/MinecraftServer.java
 +++ b/net/minecraft/server/MinecraftServer.java
-@@ -523,6 +523,7 @@
+@@ -130,7 +130,7 @@
+    private final DataFixer dataFixer;
+    private String hostname;
+    private int serverPort = -1;
+-   public WorldServer[] worlds;
++   public WorldServer[] worlds = new WorldServer[0];
+    private PlayerList playerList;
+    private boolean serverRunning = true;
+    private boolean serverStopped;
+@@ -148,7 +148,8 @@
+    private int buildLimit;
+    private int maxPlayerIdleMinutes;
+    public final long[] tickTimeArray = new long[100];
+-   public long[][] timeOfLastDimensionTick;
++   //public long[][] timeOfLastDimensionTick;
++   public java.util.Hashtable<Integer, long[]> worldTickTimes = new java.util.Hashtable<Integer, long[]>();
```

Comment:
Can we change the hash table to concurrent or regular hash map?

Reply:
Patches haven't been reviewed at this stage, I'm just implementing the minimal set to get the project compiling.